WTWSMS/events/WtWSMS_zusk_misc_events.txt

```diff
@@ -537,7 +550,7 @@ character_event = {
 			trait = sympathy_christendom
 		}
         modifier = {
-			factor = 5
+			factor = 1.05
```

Comment:
No. Originally, characters in that line that were not Hellenic or Christian were five times _**less**_ likely to get the Roman Nostalgia Event. I changed it so that they are only five percent less likely to get it.